sandbox/README.md

```diff
@@ -0,0 +1,7 @@
+This is a sandbox environment used to test react stuff in isolation.
```

Comment:
Just curious, how does this vary from setting up a form in Storybook/ were there problems with that approach?